diff --git a/node_modules/@nx/js/src/utils/inline.js b/node_modules/@nx/js/src/utils/inline.js
index ba761f4..0b5fadf 100644
--- a/node_modules/@nx/js/src/utils/inline.js
+++ b/node_modules/@nx/js/src/utils/inline.js
@@ -48,6 +48,7 @@ function postProcessInlinedDependencies(outputPath, parentOutputPath, inlineGrap
                 destDepOutputPath + '/src';
         }
     }
+    (0, fs_extra_1.removeSync)(outputPath + "/libs");
     updateImports(outputPath, inlinedDepsDestOutputRecord);
 }
 exports.postProcessInlinedDependencies = postProcessInlinedDependencies;
@@ -159,7 +160,6 @@ function movePackage(from, to) {
     if (from === to)
         return;
     (0, fs_extra_1.copySync)(from, to, { overwrite: true });
-    (0, fs_extra_1.removeSync)(from);
 }
 function updateImports(destOutputPath, inlinedDepsDestOutputRecord) {
     const importRegex = new RegExp(Object.keys(inlinedDepsDestOutputRecord)
